geomesa-core/src/test/resources/process/temporaldensity/temporaldensity.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?><wps:Execute version="1.0.0" service="WPS" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.opengis.net/wps/1.0.0" xmlns:wfs="http://www.opengis.net/wfs" xmlns:wps="http://www.opengis.net/wps/1.0.0" xmlns:ows="http://www.opengis.net/ows/1.1" xmlns:gml="http://www.opengis.net/gml" xmlns:ogc="http://www.opengis.net/ogc" xmlns:wcs="http://www.opengis.net/wcs/1.1.1" xmlns:xlink="http://www.w3.org/1999/xlink" xsi:schemaLocation="http://www.opengis.net/wps/1.0.0 http://schemas.opengis.net/wps/1.0.0/wpsAll.xsd">
```

Comment:
I tried copying the license comment found in geomesa-unique.xml / wps-unique.xml and kept getting [this error message.](https://gist.github.com/kevinwheeler/950d781e225f95dd8bf4) when trying to use this xml file to make a wps request to geoserver. Do you want me to add it anyways? Or I could delete it from both of the *-unique xml files.